### PR TITLE
Remove most of libs content and moved it to node_modules.

### LINKED ISSUE #3: Purpose of files in libs directory
What's a purpose of files found in libs directory? Most of them aren't used directly in this project. 
Some of them are external dependencies(signalR, jQuery), however documentations suggests to configure external dependencies manually using direct sources.

### PR DESCRIPTION
**Pull request by michax**
Removed most of libs content and moved it to node_modules.
jquery.signalR remains in libs as there is not npm module for it.

Only remaining reference to jquery.signalR is in tests: 
* jasmine-conf.js (vendor)

Fixes #3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,9 +28,12 @@
     "grunt-shell": "1.2.1",
     "grunt-template-jasmine-istanbul": "0.4.0",
     "ink-docstrap": "1.2.1",
+    "jquery": "^2.1.4",
     "jsdoc-75lb": "3.5.4",
     "remap-istanbul": "0.5.1",
+    "requirejs": "^2.3.3",
     "rollup-plugin-babel": "2.4.0",
-    "taffydb": "^2.7.3"
+    "taffydb": "^2.7.3",
```
Comment (lukeapage) [inline package.json]: Actually can you make these exact matches?

Comment (michax) [inline package.json]: Done

**Comment by michax** [comment]
Rebased master onto this branch and fixed shrinkwrap which on master had invalid project name.

File: package.json (hunk 1 above)
Comment (michax) [inline package.json]: In additon, I've fixed shrinkwrap, which had invalid project name on master.

**Comment by lukeapage** [comment]
@michax please rebase and then I will merge. Thanks!

**Comment by michax** [comment]
@lukeapage Done.

**Pull request merged**